.buildkite/scripts/steps/functional/cloud_security_posture_serverless.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+source .buildkite/scripts/steps/functional/common.sh
+
+export JOB=kibana-cloud-security-posture-serverless-cypress
+export KIBANA_INSTALL_DIR=${KIBANA_BUILD_LOCATION}
+
+echo "--- Cloud Security Posture Workflows Cypress tests on Serverless"
+
+cd x-pack/plugins/security_solution
+
+set +e
+
+yarn cypress:run:qa:serverless:cloud_security_posture; status=$?; yarn junit:merge || :; exit $status
```

Comment:
The command that uses the `qa` is the one for MKI, for CI it should be: `cypress:cloud_security_posture:run:serverless`